docs/cmd/prompt.md

```diff
@@ -1,57 +1,66 @@
 ## Description
-The `[prompt]` command starts an interactive gnmic prompt in which the same gNMI commands (`capabilities`, `get`, `set`, `subscribe`) can be executed with some additional auto completion features:
+The `prompt` command starts `gnmic` in an interactive prompt mode with the following auto-completion features:
 
-* Auto-completion for all commands names and their flags.
-* Auto-completion for flags with xpath values ( `--path`, `--prefix`, `--model`,...). The xpaths are generated from models suplied using `prompt` command `--file` and `--dir` (optional) flags.
-* Auto-completion for flags with enum values `--format`, `--encoding`, ...
-* Auto-completion for flags taking file or directory values.
+* All `gnmic` [commands names and their flags are suggested](../advanced/prompt_suggestions.md#commands-and-flags-suggestions).
+* Values for the flags that rely on YANG-defined data (like `--path`, `--prefix`, `--model`,...) will be dynamically suggested, we call this feature [YANG-completions](../advanced/prompt_suggestions.md#yang-completions).  
+The auto-completions are generated from the YANG modules d with the `--file` and `--dir` flags.
+* Flags with the fixed set of values (`--format`, `--encoding`, ...) will get their [values suggested](../advanced/prompt_suggestions.md#enumeration-suggestions).
+* Flags that require a [file path value will auto-suggest](../advanced/prompt_suggestions.md#file-path-completions) the available files as the user types.
 
 
 ### Usage
 
 `gnmic [global-flags] prompt [local-flags]`
 
-
-### Flags 
+### Flags
 
 #### file
-
-File or directories pointing to yang modules from which to generate gnmi paths.
+A path to a YANG file or a directory with YANG files which `gnmic` will use to generate auto-completion for YANG-defined data (paths, models).
 
 Multiple `--file` flags can be supplied.
 
 #### dir
+A path to a directory which `gnmic` would recursively traverse in search for the additional YANG files which may be required by YANG files specified with `--file` to build the YANG tree.
 
-Direcotries where to search for yang modules included and imported in modules under `--file`
+Can also point to a single YANG file instead of a directory.
 
 Multiple `--dir` flags can be supplied.
 
-#### exclude
+#### description-with-prefix
+When set, the description field of the suggestion box will have a module prefix name before the element description.
```

Comment:
let's specify this is for xpath completion only

Reply:
reworked as
> When set, the description of the path elements in the suggestion box will contain module's prefix.
in a0d15077bf6f8e91f99d53aa7f65599e3a72e450